include/modelstate.h

```diff
@@ -50,16 +50,16 @@ class ModelState
    bool setState(double *,double*,double*);
 
    //void setBaseFrameLink(int);
-   void setReferenceFrameLink(std::string);
-   void setReferenceToWorldFrameRotoTrans(wbi::Frame);
+//    void setReferenceFrameLink(std::string);
+//    void setReferenceToWorldFrameRotoTrans(wbi::Frame);
```

Comment:
@iron76 Sorry fra it was meant to be an immediate push for Silvio. I agree completely and will get all commented lines out. 
@traversaro tests - lets discuss quickly before I push my next commit including matlab